src/app/multi-editor/multi-editor.component.ts

```diff
@@ -219,11 +222,11 @@ export class MultiEditorComponent implements OnInit {
       },
       (error) => {
         if (error.json().message) {
-          this.totalRecords = -1;
-          this.errorMessage = error.json().message;
+          this.toastr.error(error.json().message);
```

Comment:
ditto

Reply:
also it seems like showing error message can be a function and used here and above.